frontend/src/components/PostCategories.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const PostCategories = ({ categories }) => {
+  return categories.map((category, index) => (
+    <p key={index}>{category}</p>
+  ));
```

Comment:
If category got id, use it as key.
```suggestion
  return categories.map((category) => (
    <p key={category.id}>{category}</p>
  ));
```

Reply:
Categories comes as an array of strings, so no Ids, I'm afraid.